### DISCUSSION TITLE
Did you make any changes to F-Droid repository name?

### DISCUSSION THREAD

**Discussion by jcperil** [Droid-ify/client#648] [Q&A] [answered]
After syncing today, the F-Droid repo name has been changed to **My First F-Droid Repo Demo**.

![droidify](https://github.com/Droid-ify/client/assets/89805719/70ac561c-302a-43eb-b262-c32bc2e66115)

Is this intended? Skeptical about this. Can you please clarify?

**Comment by Iamlooker** [comment]
I didn't, I will look into this

**Comment by TheLastProject** [comment, marked as answer]
The f-droid.org team messed up and changed the repo name: https://github.com/Droid-ify/client/issues/653#issuecomment-1913365896

Droid-ify is behaving correctly by showing the name of the repo that the repo sends.

**Comment by JokerQyou** [comment]
According to the F-Droid team, using V2 repo format should fix this issue, and it seems like they would not like to fix the name for V1 repo, see https://gitlab.com/fdroid/fdroiddata/-/issues/3173 . Is Droid-ify using the V2 format already? I see the demo name again today after syncing.

**Reply by Iamlooker** [reply]
We will start using it after sometime. I don't have time to work on this right now
